src/model/guild/guild_id.rs

```diff
@@ -21,14 +19,19 @@ use crate::collector::{
 use crate::http::{CacheHttp, Http};
 #[cfg(feature = "model")]
 use crate::internal::prelude::*;
-use crate::model::prelude::*;
+#[cfg(feature = "model")]
+use crate::json::json;
 #[cfg(feature = "model")]
 use crate::utils;
 #[cfg(all(feature = "model", feature = "unstable_discord_api"))]
 use crate::{
     builder::CreateInteraction,
     model::interactions::{ApplicationCommand, Interaction},
 };
+use crate::{
+    json::{from_number, NULL},
+    model::prelude::*,
+};
```

Comment:
```suggestion
use crate::{
    json::{from_number, NULL},
    builder::CreateInteraction,
    model::{
        prelude::*,
        interactions::{ApplicationCommand, Interaction},
    },
};
```

Reply:
Those two can not be combined since the first one depends on the `unstable_discord_api` feature the second one doesn't

Reply:
Ah, didn't pay attention to that. This is fine then.